content/java/concepts/strings/terms/getChars.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.getChars()'
+Description: ' Copies characters from the given string into the destination character array.'
+Subjects:
+  - 'Java'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **.getChars()** string method copies a set of characters from a string into a destination character array. 
+
+## Syntax
+
+The basic syntax of `.getChars()` in Java is:
+```pseudo
+getChars(int srcBegin, int srcEnd, char[] dst, int dstBegin)
+```
+A brief description of the parameters is as folllows:
+
+- `srcBegin`: The index to start copying the new array.
+- `scrEnd`: The index to stop copying the new array.
+
+dst: - 'The name of the new array. '
+dstBegin: The index of the destination array.
+
+## Example
+
+In the following example, `getChar()` converts an array into a new shorter array. It then prints the new array in the console 
+
+```class example{
```

Comment:
Again, you need to replace the example with the following:

````
```java
class Example { 
    public static void main(String[] args) { 
        String str = "Welcome to Texas"; 
        char[] destArray = new char[5]; 
        try { 
            str.getChars(0, 5, destArray, 0); 
            System.out.println(destArray); 
        } 
        catch (Exception ex) { 
            System.out.println(ex); 
        } 
    } 
}
```

This results in the following output:

```shell
Welco
```
````